### PR TITLE
Remove experimental from GPTQ facade API

### PR DESCRIPTION
**Pull request by reuvenperetz**
## Pull Request Description:
Change GPTQ API methods by removing 'experimental' from their names.

## Checklist before requesting a review:
- [ ] I set the appropriate labels on the pull request.
- [ ] I have added/updated the release note draft (if necessary).
- [ ] I have updated the documentation to reflect my changes (if necessary).
- [ ] All function and files are well documented. 
- [ ] All function and classes have type hints.
- [ ] There is a licenses in all file.
- [ ] The function and variable names are informative. 
- [ ] I have checked for code duplications.
- [ ] I have added new unittest (if necessary).

### CODE REVIEW

File: model_compression_toolkit/gptq/keras/quantization_facade.py [hunk 1]
```diff
@@ -113,15 +113,15 @@ def get_keras_gptq_config(n_epochs: int,
...
-                                                               new_experimental_exporter: bool = True) -> Tuple[Model, UserInformation]:
+    def keras_gradient_post_training_quantization(in_model: Model,
+                                                  representative_data_gen: Callable,
+                                                  gptq_config: GradientPTQConfigV2,
+                                                  gptq_representative_data_gen: Callable = None,
+                                                  target_kpi: KPI = None,
+                                                  core_config: CoreConfig = CoreConfig(),
+                                                  fw_info: FrameworkInfo = DEFAULT_KERAS_INFO,
+                                                  target_platform_capabilities: TargetPlatformCapabilities = DEFAULT_KERAS_TPC,
+                                                  new_experimental_exporter: bool = True) -> Tuple[Model, UserInformation]:
```
Comment (Idan-BenAmi) [inline model_compression_toolkit/gptq/keras/quantization_facade.py:124]: Just confirm the new_experimental_exporter should be kept experimental

File: tests/keras_tests/feature_networks_tests/base_keras_feature_test.py [hunk 2]
```diff
@@ -16,7 +16,7 @@
 from model_compression_toolkit.core.common.framework_implementation import FrameworkImplementation
 from model_compression_toolkit.target_platform_capabilities.constants import DEFAULT_TP_MODEL
 from model_compression_toolkit.core.keras.default_framework_info import DEFAULT_KERAS_INFO
-from model_compression_toolkit.gptq import keras_gradient_post_training_quantization_experimental
+from model_compression_toolkit.gptq import keras_gradient_post_training_quantization
 from model_compression_toolkit.core import FrameworkInfo
 from model_compression_toolkit.ptq import keras_post_training_quantization_experimental
```
Comment (Idan-BenAmi) [inline tests/keras_tests/feature_networks_tests/base_keras_feature_test.py:21]: experimental.. should be removed?

**Review by Idan-BenAmi — APPROVED** [review]

File: tests/keras_tests/feature_networks_tests/base_keras_feature_test.py (hunk 2 above)
Comment (reuvenperetz) [reply tests/keras_tests/feature_networks_tests/base_keras_feature_test.py:21]: PTQ is handled in a different PR
